src/ProtocolZoo/ProtocolZoo.jl

```diff
@@ -389,6 +396,8 @@ $FIELDS
     nodeB::Int
     """time period between successive queries on the nodes (`nothing` for queuing up and waiting for available pairs)"""
     period::LT = 0.1
+    """tag for which the consumer is consuming"""
```

Comment:
```suggestion
    """tag type which the consumer is looking for -- the consumer query will be `query($node, EntanglementConsumer.tag, $remote_node)` and it will be expected that `$remote_node` possesses the symmetric reciprocal tag; defaults to `EntanglementCounterpart`"""
```